accesscontroltool-package/src/main/META-INF/vault/filter.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-	<filter root="/apps/netcentric/actool"/>       
+	<filter root="/apps/netcentric/actool"/>  
+	<filter root="/home/users/system/netcentric/actool/acl/ynUbs6nnylcpRBs0eXdv"/>
+	<filter root="/" mode="merge"/>

```

Comment:
This potentially can take veeery long to install (depending on how much content the instance has). Do we have a chance to set it to /rep:policy or similar? If not, the ACE will have to be added by a an install hook (custom code) or repo init (or whatever else can do it)